src/MOI_wrapper.jl

```diff
@@ -134,6 +134,19 @@ function _allocate_constraint(cone::ConeData, f, s::MOI.ExponentialCone)
     cone.ep += 1
     ci
 end
+constroffset(cone::ConeData, ci::CI{<:MOI.AbstractFunction, <:MOI.PowerCone}) = cone.f + cone.l + cone.q + cone.s + cone.ep + ci.value
```

Comment:
Exceeds 80 character limit: see the [JuMP style guide](http://www.juliaopt.org/JuMP.jl/dev/style/)